libs/core/langchain_core/prompts/chat.py

```diff
@@ -922,7 +935,7 @@ class ChatPromptTemplate(BaseChatPromptTemplate):
 
     """  # noqa: E501
 
-    messages: List[MessageLike]
+    messages: Annotated[List[MessageLike], SkipValidation]
```

Comment:
used SkipValidation() in other places, dunno if that matters

Reply:
oh good catch! I wonder why this works